### PR TITLE
Number text components

### PR DESCRIPTION
**Pull request by kappu72**

### CODE REVIEW

File: web/client/components/data/query/SimpleFilterField.jsx [hunk 1]
```diff
@@ -9,21 +9,24 @@
 const React = require('react');
 const {Panel, ButtonToolbar, Button, OverlayTrigger, Tooltip} = require('react-bootstrap');
 const ComboField = require('./ComboField');
+const NumberField = require('./NumberField');
+const TextField = require('./TextField');
 const {isEqual} = require('lodash');
 
 const SimpleFilterFiled = React.createClass({
```
Comment (mbarto) [inline web/client/components/data/query/SimpleFilterField.jsx]: SimpleFilterField

File: web/client/components/data/query/SimpleFilterField.jsx [hunk 2]
```diff
@@ -9,21 +9,24 @@
...
         optionsValues: React.PropTypes.array,
         defaultOptions: React.PropTypes.array,
         optionsLabels: React.PropTypes.object,
-        values: React.PropTypes.array,
+        values: React.PropTypes.oneOfType([React.PropTypes.array, React.PropTypes.object, React.PropTypes.number, React.PropTypes.string]),
         multivalue: React.PropTypes.bool,
         combo: React.PropTypes.bool,
         toolbar: React.PropTypes.bool,
-        type: React.PropTypes.oneOf(['list', 'date', 'bool']),
+        type: React.PropTypes.oneOf(['list', 'number', 'string']).isRequired,
```
Comment (mbarto) [inline web/client/components/data/query/SimpleFilterField.jsx]: Why did you remove 'date' and 'bool'?

File: web/client/components/data/query/SimpleFilterField.jsx (hunk 1 above)
Comment (kappu72) [inline web/client/components/data/query/SimpleFilterField.jsx]: Sorry for the typo.

File: web/client/components/data/query/SimpleFilterField.jsx (hunk 2 above)
Comment (kappu72) [inline web/client/components/data/query/SimpleFilterField.jsx]: I'll remove date and bool because they aren't implemented and removing them will result in a warning from React-PropTypes validator if passed as option.
